### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by PeterChaplin**
Not ready for merge yet - just a place for comments.

### CODE REVIEW

File: docs/cycleText.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Text cycle plugin
+## $.fn.cycleText( labels )
+[Fiddle](http://jsfiddle.net/PeterChaplin/6nuMD/)
+
+Call on a single element with text content matching one of the labels provided (otherwise behavior is unspecified).
+Cycles through the given labels (each call replaces the current contents of the element with the next label).
+When the end of the array is reached, cycle back to the first one.
+
+### labels
+Type: Ayyar (required)
```
Comment (bvw) [inline docs/cycleText.md]: typo ? is that Array ?

Comment (PeterChaplin) [inline docs/cycleText.md]: Yes.

**Comment by bvw** [comment]
Would it be okay to merge at this stage just to keep the size of the changes manageable ?
You could keep working on the same branch.

**Comment by PeterChaplin** [comment]
Yes, I think so.

**Pull request merged**